### DISCUSSION TITLE
Make Weblate ignore a msgid, treat it read-only or untranslatable?

### DISCUSSION THREAD

**Discussion by buhtz** [WeblateOrg/weblate#9599] [Q&A]
My po-files will have on msgid/msgstr at the end like this:

    msgid = "__COMPLETENESS__"
    msgstr = "'{\"ja\": 30.5, \"de\": 71}'"

Can I make weblate to somehow "ignore" that entry so that the translators won't see it or won't be able to manipulate it?

**Comment by nijel** [comment]
Just add `read-only` flag to that string in Weblate. If you have more of them, https://docs.weblate.org/en/latest/admin/addons.html#bulk-edit can be used.

**Reply by buhtz** [reply]
Do you mean like this?

```
#, read-only
msgid = "__COMPLETENESS__"
msgstr = "'{\"ja\": 30.5, \"de\": 71}'"
```

**Reply by nijel** [reply]
That would work as well (but will probably cause you some issues as gettext doesn't like flags it doesn't know), but I meant in Weblate, see https://docs.weblate.org/en/latest/admin/checks.html#customizing-behavior-using-flags

**Reply by buhtz** [reply]
Can you please give me an example based on my example to explain your solution?
The links you provided to not help me.

**Reply by nijel** [reply]
Edit the flags in the user interface: https://docs.weblate.org/en/latest/admin/translating.html#additional

**Reply by buhtz** [reply]
No I understand the cause of misunderstanding. 😄 
I missed to tell that I don't want to use Weblate for that. I do manipulate the po files (before uploading them to weblate or let weblate pull/clone them) via my own scripts and tools.
I'll try that `#, read-only` in the po-file and will see how my toolchain and weblate react on it.
